api/include/opentelemetry/logs/logger.h

```diff
@@ -35,6 +41,17 @@ namespace logs
  **/
 class Logger
 {
+protected:
+  // Default values to set for fields that the user doesn't specify
+  core::SystemTimestamp default_timestamp_;  // uint64 nanoseconds since Unix epoch, default 0
+  Severity default_severity_;                // default 0 (kInvalid)
+  nostd::string_view default_name_;          // default 0 length, nullptr data
+  nostd::string_view default_body_;  // default 0 length, nullptr data. TODO: currently a simple
+                                     // string, but should be changed "Any" type
+  trace::TraceId default_trace_id_;  // default 00000000000000000000000000000000
```

Comment:
Are these numbers in in hex? Better to have prefix like `0x` to make the indication clear.

Reply:
Yeah `0x00000000000000000000000000000000` would be the value of the TraceId if it were converted to hex. But internally Do we refer to TraceIds or SpanIds in Hex? (i.e. or just simple as 

Internally, it is a 32 `uint8_t` array all initialized to `0`s, but it would be converted to lowercase base 16 representation before being used. 

Would it be simpler to say "default value all initialized to 0s"?

Reply:
Does this follow the convention for Trace/SpanIds? Or would it simply be better to say `default 00000000000000000000000000000000, in base 16`

Reply:
Yes, the suggestion looks good too.